dcpy/library/templates/dycd_service_sites.yml

```diff
@@ -0,0 +1,37 @@
+dataset:
+  name: &name dycd_service_sites
+  version: "{{ version }}"
```

Comment:
Don't need this line anymore.

Actually a few things with latest main are not needed
- no version if not actually specified
- no &name in `name` field
- no name field in destination (assumes its the `name` of the dataset. Before, there was validation that would throw an error if they didn't match, so I just got rid of it entirely)